src/frontend/packages/lib_video/package.json

```diff
@@ -0,0 +1,124 @@
+{
+  "name": "lib-video",
+  "version": "1.0.0",
+  "description": "",
+  "license": "ISC",
+  "main": "lib/index.js",
+  "module": "lib/index.es.js",
+  "jsnext:main": "lib/index.es.js",
+  "scripts": {
+    "build": "rollup -c && yarn tsc-alias ",
```

Comment:
Adding tsc-alias after the rollup build breaks the rollup watch option.
Here the commit fixing this problem: https://github.com/openfun/marsha/commit/991580e35d924b808e61fee6b546b723e50bc5d4